.github/workflows/publish_develop.yml

```diff
@@ -53,8 +53,18 @@ jobs:
           fi
           echo "TAG=$TAG" >> $GITHUB_ENV
       - name: Build and push image
+        if: github.ref == 'refs/heads/develop'
+          echo 'Building image from the ${{ github.ref }} branch'
```

Comment:
are you sure this syntax works? can we execute an arbitrary bash command in the `if` block of the action?

Reply:
> @Emmanuel289 see my comments above ☝️ also it looks like you need to resolve come conflicts

thanks. I will work on the comments and update shortly